libclc/utils/libclc-remangler/CMakeLists.txt

```diff
@@ -25,6 +25,24 @@ clang_target_link_libraries(libclc-remangler
   LLVMOption
   )
 
+if(CMAKE_CROSSCOMPILING)
+  if(LLVM_NATIVE_TOOL_DIR AND NOT LIBCLC_REMANGLER_PATH)
+    if(EXISTS "${LLVM_NATIVE_TOOL_DIR}/libclc-remangler${LLVM_HOST_EXECUTABLE_SUFFIX}")
+      set(LLVM_CONFIG_PATH "${LLVM_NATIVE_TOOL_DIR}/libclc-remangler${LLVM_HOST_EXECUTABLE_SUFFIX}")
```

Comment:
Should this be setting `LIBCLC_REMANGLER_PATH`?

Reply:
Yes it should've but I have decided to not do all the cross compiling logic manually, instead using `setup_host_tool`.